Packs/ANYRUN/Integrations/ANYRUN/ANYRUN.py

```diff
@@ -878,6 +878,8 @@ def run_analysis(args):
             args['env_version'] = '10'
         else:
             args['env_version'] = '7'
+        if obj_type == 'file' or obj_type == 'download' :
+            args['obj_ext_cmd'] = args.get('obj_ext_cmd')
```

Comment:
```suggestion
        if obj_ext_cmd and (obj_type == 'file' or obj_type == 'download'):
            args['obj_ext_cmd'] = args.get('obj_ext_cmd')
```
and add this :
`obj_ext_cmd = args.get('obj_ext_cmd')`
in line 854 under:
`obj_type = args.get('obj_type')`